server/client_manager.py

```diff
@@ -510,15 +512,32 @@ def __init__(self, server):
         self.server = server
         self.cur_id = [i for i in range(self.server.config['playerlimit'])]
 
+
+    def new_client_auth(self,transport): #future authentication methods should go here
```

Comment:
This method does not have a great name, considering there are other reasons a client might be rejected, such as the ban check.